.github/CODEOWNERS

```
@@ -1 +1,3 @@
-* @tianon @yosifkit
+*  @docker-library/maintainers
+
+/doi.jq             @LaurentGoderre  @docker-library/maintainers
```

A lot of extra whitespace here :sweat_smile: (we usually only do this when we're lining up columns in multiple lines)

```suggestion
/doi.jq  @LaurentGoderre  @docker-library/maintainers
```

Also, note GitHub's :no_entry: icon -- we're missing newline-at-EOF :eyes:

---

Done!